src/serialization.py

```diff
@@ -0,0 +1,237 @@
+"""
+File Name: serialization.py
+
+Copyright (c) 2023 - 2024 IndoorJson
+
+Author: Ziwei Xiang <knightzz1016@gmail.com>
+Create Date: 2024/3/13
+"""
+
+import numpy as np
+from shapely.wkt import loads
+
+
+class Graph:
```

Comment:
If the data structure is not only about the graph but also about cellspace in primal space, then the name is not good.